internal/graph/urn.go

```diff
@@ -3,10 +3,15 @@ package graph
 import (
 	"errors"
 	"fmt"
+	"github.com/gleanerio/nabu/pkg/config"
+	"github.com/spf13/viper"
 	"strings"
 )
 
-func MakeURN(s, bucket string) (string, error) {
+// MakeURN formats a URN following the ADR 0001-URN-decision.md  which at the
+// time of this coding resulted in     urn:{program}:{organization}:{provider}:{sha}
+func MakeURN(v1 *viper.Viper, s, bucket string) (string, error) {
+	gcfg, _ := config.GetGleanerConfig(v1)
```

Comment:
trying a 'release' and fails.

Reply:
To make the URN with the organization I had to put 

```yaml
gleaner:
  runid: iow # this will be the bucket the output is placed in...
  summon: true # do we want to visit the web sites and pull down the files
  mill: false
```

into the nabu config.  

My next TODO with GleanerIO is to make the gleaner and nabuy configs the same file.